index.html

```diff
@@ -4444,8 +4453,19 @@ <h3>Parsing content</h3>
       a |record:NDEFRecord|, run these steps:
       <ol class=algorithm data-link-for="NDEFRecord">
         <li>
-          Set |record|'s <a>recordType</a> to the
-          value of |ndefRecord|'s <a>TYPE field</a>.
+          Let |domain| and |type| be the result of running [=split external type=] with the value of |ndefRecord|'s <a>TYPE field</a>.
```

Comment:
We should return null in cases of failure so that the record is not added to message's records.

Reply:
In this case, they have been validated earlier so they will always be valid

Reply:
Where have they been validated earlier?